### ISSUE TITLE
Changes to stan models in source not recognized -- "make: Nothing to be done for 'all'."

### ISSUE THREAD

**Issue by cdriveraus** [stan-dev/rstantools#124]
When building ctsem (using rstantools), I'm encountering this all of a sudden -- after some small changes to the stan files, I run rstan_config() and double check that the source files are changed, but compilation is for some reason not triggered. If instead I delete the relevant source files and build, compilation works normally and everything is as expected. Any ideas welcome...

**Comment by helske** [comment]
Old issue, but it seems that the issues persists, I've encountered this several times lately using latest `rstantools` etc when using clean and install button on Rstudio.

**Comment by mcol** [comment]
It would have to have some reproducible steps, or in any case at least on what operating system this happens.

**Comment by helske** [comment]
Yeah this is very vague, I'm on Windows 10. I'm not 100% sure what just happened, but I think after I manually called `rstantools::rstan_config()` nothing changed, but then after restarting R recompilation was triggered. Perhaps this is actually some kind of cache issue related to Rcpp. I'll try to keep track of what I'm doing more carefully for the next time this happens.
